src/hooks/useSubscriptions.ts

```diff
@@ -41,18 +41,17 @@ export function useConfigSubscription(garden: Garden | null) {
 
     configSubscription.current = garden.onConfig(onConfigHandler)
 
-    return () => {
-      configSubscription.current = null
-    }
+    return () => configSubscription.current.unsubscribe()
```

Comment:
This should be:

```suggestion
    return () => {
      configSubscription.current.unsubscribe()
    }
```

because useEffect expects undefined during unsubscribing.